docs/contribute/contribute.adoc

```diff
@@ -4,126 +4,169 @@ This document is meant for new OpenSCAP contributors and it describes how to
 contribute to the OpenSCAP project. Let's suppose that you already have an
 active Github's account with a user name _adam_ and you want to fix one of the
 issue from the link:https://github.com/OpenSCAP/openscap/issues[tracker]. Let's
-say that we want to fix the isuues with number _455_ and the fix will go into
-the _maint-1.0_ branch.
+say that you want to fix the isuues with number _455_ and the fix will go into
+the `maint-1.2` branch.
+
+NOTE: This guide also applies for adding a new feature, the process is the same
+as for fixing an issue described below.
+
 
 == Fork the OpenSCAP repository
 Before you start working on a fix it's a good practice to leave a
 comment in the issue that you work on the fix so other contributors know that
 the fix is in progress.  Next thing you have to do is to fork the OpenSCAP
-repository. You can do that by pressing the **Fork** button in the top-right
+repository. You can do that by pressing the *'Fork'* button in the top-right
 corner on the Github's link:https://github.com/OpenSCAP/openscap[OpenSCAP page].
 It will create a copy of the original repository which you can use for
 proposing changes.
 
+
 == Choose the right branch
-Before you start working of the fix it's necessary to determine which branch the
+Before you start working on the fix it's necessary to determine which branch the
 fix will go into. If you are not familiar with the OpenSCAP's branches or
 versions yet please have a look at the link:versioning.adoc[versioning]
 document. Be aware that the fact that an issue description says that the fix
-should go to a _maint-1.2_ branch doesn't have to be true. It's a good practice
+should go to the `maint-1.2` branch doesn't have to be true. It's a good practice
 to investigate the correct branch first or ask experienced developers on our
 FreeNode IRC channel called `#openscap` or
 link:https://www.redhat.com/mailman/listinfo/open-scap-list[mailing list].
 
-Once you have forked the repository and decided what branch the fix will go into
-you can clone your forked repository.
+NOTE: The default branch of the openscap repository is set to the `maint-1.2`.
+
+First you need to clone your forked repository:
 [[app-listing]]
 [source,bash]
 ----
 $ git clone git@github.com:adam/openscap.git
 ----
 
-
-Now you can create a new branch, which we will call _fix_455_, where you can work on the fix. Remember that
-the name of the new branch will appear in the commit message when your fix is
-merged so choose the name wisely.
+Once you have forked the repository and decided that the fix will go into the
+`maint-1.2` branch you can create a new branch from it, which we will call
+_fix_455_, where you can work on the fix. Remember that the name of the new
+branch will appear in the commit message when your fix is merged so choose the
+name wisely:
```

Comment:
lol, I totally fail at this and provide random or funny or outright rude branch names